packages/charts/src/charts/table/table.utils.tsx

```diff
@@ -0,0 +1,31 @@
+import { rgb } from 'd3-color';
+import { SortByType } from '@keen.io/ui-core';
+
+export const firstCapital = (str: string) => {
+  return str.replace(/^\w/, (c: string) => c.toUpperCase());
+};
+
+export const hexRgb = (hex: string, opacity: number) => {
```

Comment:
The `keen.io/charts` has already `polished` in dependency. I believe the same functionality could be imported from that package.